docker/localsettings.py

```diff
@@ -118,7 +118,11 @@
     "url": "http://riakcs:9980/",
     "access_key": "admin-key",
     "secret_key": "admin-secret",
-    "config": {"connect_timeout": 3, "read_timeout": 5},
+    "config": {
+        "connect_timeout": 3,
+        "read_timeout": 5,
+        "signature_version": "s3"
```

Comment:
@millerdev remember this bug? Looks like we could have caught it if we had fixed our docker build sooner :/